docs/contributing/Masterlist-Editing.html

```diff
@@ -90,65 +109,115 @@ <h3>Common Metadata</h3>
+    msg:
+      - &lt;&lt;: *useBashTweakInstead
+        subs: [ 'Tweak Actors -> Quiet Feet' ]
+  - name: 'Horse Management.esp'
+    url: [ 'https://www.nexusmods.com/oblivion/mods/46021' ]
+    req: [ *OBSEv21 ]</code></pre>
 
 <p>Notice how in the example above, the <code>common</code> node has two different types of data structure in the same list (message and file structures). If this was done anywhere in the <code>globals</code> or <code>plugins</code> nodes LOOT would complain, because it expects a certain format, but because LOOT doesn't look at the <code>common</code> node, this is OK.</p>
 
 <h3>Dirty Edit Metadata</h3>
 
-<p>If a user posts dirty counts for a plugin that already has a dirty message for the same CRC, and the counts are different to what's in the masterlist, just replace the existing counts with what the user gave if they used the latest version of TES5Edit. If they didn't give the TES5Edit version number, only replace the existing counts if the new counts are higher (which generally indicates a newer version).</p>
+<p>If a user posts dirty counts for a plugin that already has a dirty message for the same CRC, and the counts are different to what's in the masterlist, just replace the existing counts with what the user gave if they used the latest version of xEdit (TES4Edit, SSEEdit, FO4Edit, etc). Also make sure that xEdit's QAC (Quick Auto Clean) feature was used to generate the cleaning reports.</p>
 
 <h4>Intentional ITMs</h4>
 
-<p>A very tiny minority of dirty edits (and only ever ITMs) are intentional and should not be removed - this is for cases where a mod will not function correctly unless certain data have their vanilla values, so the mod includes a copy of the vanilla data to override anything that might change it.</p>
+<p>Sometimes a mod author is of the opinion, that the ITMs in a plugin are "Intentional". Since this topic is not viewed in the same way by everyone, we have a few options to handle this situation. For one, if the plugin in question also has "Deleted References" or "Deleted Navmeshes", we should continue to include the xEdit cleaning report within our masterlists. If the plugin in question <i>only</i> includes "Intentional" ITMs, we normally would still feature the xEdit cleaning report in our masterlists. But if the mod author of that plugin would prefer that those ITMs shouldn't be cleaning because of their intentional nature, we can remove the cleaning report from the masterlist and add a "#doNotClean" comment to the plugin. We will however not advise our users to not clean those plugins, not reporting these "Intentional" ITMs is the most we can offer mod authors in this situation.</p>
 
-<p>Mod authors who don't have a good understanding of dirty edits will often claim that their edits are required - you must download the mod in question and see for yourself if any such claim is made.</p>
+<p>It is however always best to double check those plugins with the latest version of xEdit.</p>
```

Comment:
Grammatical suggestion

```suggestion
<p>It is, however, always best to double check those plugins with the latest version of xEdit.</p>
```